solutions/community-deploy/cloud/AWS/DEPLOY.md

```diff
@@ -0,0 +1,80 @@
+# Setup K8s Cluster and Deploy CORTX Stack
+
+   Please find below steps to setup K8s cluster and deploy CORTX Stack.
+
+## Prerequisite 
+### Minimum specification required for Cortx Stack
+   - RAM: 16GB
+   - CPU: 8Core
+   - DISK: Need 8 Disk with 10GB per disk.
+
+### If you have automated deployment for solution yaml, please make sure your vm having below drives available
+```
+      ls /dev/sd*
+```
+   - `/dev/sdb  /dev/sdc  /dev/sdd  /dev/sde  /dev/sdf  /dev/sdg  /dev/sdh  /dev/sdi  /dev/sdj`
+
+### If Git doesn't exist in your system then run below command to install it
```

Comment:
Please change this to - 
`Git package should be installed on the system. You can use below command to install it`

Reply:
Changed above line